caffe2/contrib/aten/gen_op.py

```diff
@@ -153,6 +153,11 @@ def supports(o, factory_methods):
             print("Skipping {} Because of Arg: {} ({}) ".format(
                   o['name'], arg['type'], arg['dynamic_type']))
             return False
+
+    # skip _foreach_clamp(Tensor[], Scalar?, Scalar?)
+    if o['name'] in {"_foreach_clamp"}:
+        print(f"Skipping {o['name']} because it has multiple scalar arguments with default values.")
+        return False
```

Comment:
I have no idea but somehow in the generated file `_foreach_clamp` is called only with one argument w/o min and max specified.

Reply:
cc @bdhirsh is this patch okay or is there a more systematic way to have the scalar args be represented?

Reply:
hmmm I'm not familiar with what this file does (looks like some very old codegen from caffe2).

But given that this change won't be BC breaking (since you're just adding a brand new op and excluding it from this codegen), this change seems fine to me.